README.md

```diff
@@ -177,6 +178,29 @@ pip3 install p4runtime
 pip3 install p4runtime==1.3.0
 ```
 
+### Rust
+
+To include the P4Runtime Rust crate to your project, use the `cargo` command:
+
+```bash
+cargo add p4runtime
+```
+
+Or add the following to your `Cargo.toml` file:
+
+```toml
+[dependencies]
+p4runtime = "1.3.0"
+```
```

Comment:
I don't see how this could work, given that there is no `p4runtime` crate on crates.io
I would suggest only having the "git dependency" method below in this README, if this is the only method that works at the moment.

Reply:
Ok, we can add this intro once it is published to crates.io.

I will delete this.